kong/pdk/tracing.lua

```diff
@@ -253,13 +255,18 @@ end
 -- span:set_attribute("net.peer.port", 443)
 -- span:set_attribute("exception.escaped", true)
 function span_mt:set_attribute(key, value)
+  -- key is decided by the programmer, so if it is not a string, we should
```

Comment:
```suggestion
  -- key is a user input, so if it is not a string, we should
```

Reply:
It's not user input, that's why we can error out without worrying this would fail requests.